test/e2e/telemetry_logs_analysis_test.go

```diff
@@ -50,10 +54,17 @@ var _ = Describe("Telemetry Components Error/Warning Logs Analysis", Label("tele
 		var objs []client.Object
 		objs = append(objs, kitk8s.NewNamespace(mockNs).K8sObject())
 
+		// logs overrides (include agent logs)
+		overrides := kitk8s.NewOverrides().WithPaused(false).WithCollectAgentLogs(true)
+		objs = append(objs, overrides.K8sObject())
+
 		// backends
 		otelCollectorLogBackend := backend.New(otelCollectorLogBackendName, mockNs, backend.SignalTypeLogs)
 		objs = append(objs, otelCollectorLogBackend.K8sObjects()...)
 		otelCollectorLogTelemetryExportURL = otelCollectorLogBackend.TelemetryExportURL(proxyClient)
+		fluentBitLogBackend := backend.New(fluentBitLogBackendName, mockNs, backend.SignalTypeLogs)
```

Comment:
Why do we need two log backends?

Reply:
Because I have two separate test cases, and I want to test otelCollector-based and FluentBit-based components separately, so that we have a more accurate understanding of what components are failing, in case something goes wrong.